### ISSUE TITLE
sync 29.11.2021

### ISSUE THREAD

**Issue by kuzdogan** [sourcifyeth/org#78]
- [Twitter account] past 1k followers :partying_face: 
- Wireframes with multiple contracts
- Optimism mainnet contracts verified
  - 266 full match, 28 partial match, 22 unverified.
  - Also verified [L1 Optimism contracts](https://github.com/ethereum-optimism/optimism/tree/master/packages/contracts/deployments)
- New networks: Optimism, Boba, Velas
- Failing tests resolved :heavy_check_mark: 
  - TLDR: `.env` vars were not loading in tests

**Comment by kuzdogan** [comment]
# Discussions

# Action Items
- Receiving S3 credentials this week. Set up auto back-up.
- Setting up docs.
